Sources/AnyLint/Extensions/StringExt.swift

```diff
@@ -8,7 +8,7 @@
     /// Info about the exact location of a character in a given file.
     public typealias LocationInfo = (line: Int, charInLine: Int)
 
-    func locationInfo(of index: String.Index) -> LocationInfo {
+    public func locationInfo(of index: String.Index) -> LocationInfo {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [public declarations should be documented.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5663050)